mne/io/write.py

```diff
@@ -105,14 +105,14 @@ def write_string(fid, kind, data):
     _write(fid, str_data, kind, data_size, FIFF.FIFFT_STRING, my_dtype)
 
 
-def write_name_list(fid, kind, data):
+def write_name_list(fid, kind, data, separator=':'):
```

Comment:
I don't think we can do this actually, it won't be compatible with MNE-C or MNE-MATLAB. Do we really *need* to have `:` available?